force-app/main/default/lwc/nksKnowledgeViewCounter/nksKnowledgeViewCounter.html

```diff
@@ -0,0 +1,9 @@
+<template>
+    <template if:true={hasPermission}>
```

Comment:
Skal vi vise noe melding e.l. hvis hasPermission === false?

Reply:
Dette er noe som skal vises bare for redaktørene som har permission, ellers vises ingenting; mulig vi trenger ikke vise dem heller. Har bare laget komponenten slik vi har for nyheter